sondehub_aprs_gw/comment_telemetry.py

```diff
@@ -45,6 +45,9 @@ def extract_comment_telemetry(payload):
         if payload['aprs_tocall'] == 'APZQAP':
             return extract_RS41HUP_telemetry(payload)
 
+        if payload['aprs_tocall'] == 'APZM20':
```

Comment:
I don't see `APZM20` in https://github.com/aprsorg/aprs-deviceid/blob/main/tocalls.yaml
Won't stop merging this, but would be nice to have a "official" id